lfucg-exactions/client/js/components/AgreementForm.js

```diff
@@ -0,0 +1,219 @@
+                                                    </option>
+                                                ) : (
+                                                    <option value="choose_account" aria-label="Select an Account">
+                                                        Select an Account
+                                                    </option>
+                                                )}
+                                                {accountsList}
+                                            </select>
+                                        </div>
+                                        <div className="col-sm-6">
+                                            <FormGroup label="Resolution Number" id="resolution_number">
+                                                <input type="text" className="form-control" placeholder="Resolution Number" />
+                                            </FormGroup>
+                                        </div>
+                                    </div>
+                                    <div className="row">
+                                        <div className="col-sm-6">
+                                            <FormGroup label="Date Executed" id="date_executed">
+                                                <input type="date" className="form-control" placeholder="Date Executed" />
+                                            </FormGroup>
+                                        </div>
+                                        <div className="col-sm-6 form-group">
+                                            <label htmlFor="expansion_area" className="form-label" id="expansion_area">* Expansion Area</label>
+                                            <select className="form-control" id="expansion_area" onChange={formChange('expansion_area')} >
+                                                {agreements.expansion_area ? (
+                                                    <option value="expansion_area" aria-label={`Expansion Area ${agreements.expansion_area}`}>{agreements.expansion_area}</option>
+                                                ) : (
+                                                    <option value="choose_expansion_area" aria-label="Choose an Expansion Area">Choose an Expansion Area</option>
+                                                )}
+                                                <option value={['EA-1', 'EA-1']}>EA-1</option>
+                                                <option value={['EA-2A', 'EA-2A']}>EA-2A</option>
+                                                <option value={['EA-2B', 'EA-2B']}>EA-2B</option>
+                                                <option value={['EA-2C', 'EA-2C']}>EA-2C</option>
+                                                <option value={['EA-3', 'EA-3']}>EA-3</option>
+                                            </select>
```

Comment:
This is another case where we should probably be populating these options from an API call (like states & zip codes).  It is slightly less important in this project, since we know this list will never change, but would be best practice.